{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "/home/zpp/graduation_design/pymarl_base_HRL2/pymarl-master/src",
    "dependencies": [
      "munch==2.5.0",
      "numpy==1.19.5",
      "PyYAML==5.4.1",
      "sacred==0.8.2",
      "torch==1.7.1"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [],
    "sources": [
      [
        "main.py",
        "_sources/main_758fbd5734579733d1a65a9522fb1179.py"
      ],
      [
        "run.py",
        "_sources/run_4b5e21b368138b12af270c7daae9b850.py"
      ],
      [
        "utils/__init__.py",
        "_sources/__init___d41d8cd98f00b204e9800998ecf8427e.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_2c814fe775935963fe055f6f37191d6e.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/zpp/anaconda3/envs/Smac/lib/python3.8/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base_HRL2/pymarl-master/src/main.py\", line 35, in my_main\n    run(_run, config, _log)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base_HRL2/pymarl-master/src/run.py\", line 48, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base_HRL2/pymarl-master/src/run.py\", line 257, in run_sequential\n    action_batch, macro_batch = runner.run(test_mode=False)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base_HRL2/pymarl-master/src/runners/episode_runner.py\", line 293, in run\n    reward, terminated, env_info = self.env.step(actions[0])\n",
    "  File \"/home/zpp/anaconda3/envs/Smac/lib/python3.8/site-packages/smac/env/starcraft2/starcraft2.py\", line 396, in step\n    actions_int = [int(a) for a in actions]\n",
    "  File \"/home/zpp/anaconda3/envs/Smac/lib/python3.8/site-packages/smac/env/starcraft2/starcraft2.py\", line 396, in <listcomp>\n    actions_int = [int(a) for a in actions]\n",
    "RuntimeError: CUDA error: device-side assert triggered\n"
  ],
  "heartbeat": "2021-06-20T13:28:25.784807",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) Platinum 8280 CPU @ 2.70GHz",
    "gpus": {
      "driver_version": "460.39",
      "gpus": [
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        }
      ]
    },
    "hostname": "admin001-SYS-7049GP-TRT",
    "os": [
      "Linux",
      "Linux-5.8.0-49-generic-x86_64-with-glibc2.10"
    ],
    "python_version": "3.8.5"
  },
  "meta": {
    "command": "my_main",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2021-06-20T13:26:06.763796",
  "status": "FAILED",
  "stop_time": "2021-06-20T13:28:25.791709"
}